sourcegraph/about
# Blog/ Cloud Optimization: How Sourcegraph Helps

Closes #5607 by adding a new blogpost ([draft here](https://docs.google.com/document/d/1tAq25YIC-dvIo3NfSP7vdT36l9JIhtbeCci_Y2GOXZQ/edit#)).

### Test
1. Ensure prettier has standardized the proposed changes.
2. Navigate to [`/blog/cloud-optimization-part-1-visibility`](https://deploy-preview-5617--sourcegraph.netlify.app/blog/cloud-optimization-part-1-visibility) and ensure requirements are met.
